vhdbuilder/packer/install-dependencies.sh

```diff
@@ -16,6 +16,12 @@ VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
 
 echo "Starting build on " $(date) > ${VHD_LOGS_FILEPATH}
 
+if [[ $OS == $MARINER_OS_NAME ]]; then
+  chmod 755 /opt
+  chmod 755 /opt/azure
+  chmod 644 ${VHD_LOGS_FILEPATH}
+fi
```

Comment:
I'm curious why is Mariner different than Ubuntu here?

Reply:
Mariner has a stricter umask setting than Ubuntu by default, so when a file gets created "other" users wont have read permissions. Packer runs most of the provisioning under sudo but the file download step to grab release-notes.txt is just running as the packer user, which ends up not having access to vhd-install.complete.

I'm going to look more into the permissions on the /opt/azure directory as general users should probably have read access. I'll clean this part up in the future if I manage to smooth that out.

Reply:
I see. Thanks for the explanation.